src/utils/functions/index.js

```diff
@@ -1,8 +1,23 @@
+const { statAsync } = require('../../lib/fs')
+const { getPathInProject } = require('../../lib/settings')
+
 const getFunctionsDir = ({ flags, config }, defaultValue) =>
   flags.functions ||
   (config.dev && config.dev.functions) ||
   config.functionsDirectory ||
   (config.dev && config.dev.Functions) ||
   defaultValue
 
-module.exports = { getFunctionsDir }
+const getInternalFunctionsDir = async () => {
+  const path = getPathInProject(['functions-internal'])
+
+  try {
+    const stat = await statAsync(path)
```

Comment:
[sand] As we already have it as a dependency, can we use https://github.com/sindresorhus/path-type#isdirectorypath?

Reply:
Done in https://github.com/netlify/cli/pull/2954/commits/036573c34f19868818b94a884f24dbc8cd60d5ce.